utils_test.go

```diff
@@ -291,3 +293,82 @@ func TestIsolationLevelMapping(t *testing.T) {
 		t.Fatalf("Expected error to be %q, got %q", expectedErr, err)
 	}
 }
+
+func parseDateTimeOld(str string, loc *time.Location) (t time.Time, err error) {
```

Comment:
This was useful for demonstrating the diff, but please remove this now. We do not want to keep the old code around.

Reply:
done